src/main/antlr4/GobraLexer.g4

```diff
@@ -0,0 +1,101 @@
+lexer grammar GobraLexer;
+import GoLexer;
+
+FLOAT_LIT : DECIMAL_FLOAT_LIT | HEX_FLOAT_LIT;
+
+DECIMAL_FLOAT_LIT      : DECIMALS ('.'{_input.LA(1) != '.'}? DECIMALS? EXPONENT? | EXPONENT)
+                       | '.'{_input.index() <2 || _input.LA(-2) != '.'}? DECIMALS EXPONENT?
+                       ;
+
+// BEGIN GOBRA
+//CURLIES : '{' (CURLIES|~[{}])* '}' ;
+
+TRUE        : 'true';
+FALSE       : 'false';
+ASSERT      : 'assert';
+ASSUME      : 'assume';
+INHALE      : 'inhale';
+EXHALE      : 'exhale';
+PRE         : 'requires';
+PRESERVES   : 'preserves';
+POST        : 'ensures';
+INV         : 'invariant';
+DEC         : 'decreases';
+PURE        : 'pure';
+IMPL        : 'implements';
+OLD         : 'old';
+LHS         : '#lhs';
+FORALL      : 'forall';
+EXISTS      : 'exists';
```

Comment:
It could be circumvented by replacing all occurences of `IDENTIFIER` by a new rule 
```
identifier: OLD | EXISTS | ... | IDENTIFIER;
```
This could cause a performance hit because it might cause most files to have to be parsed with the slower, more powerful LL parser instead of the faster SLL, but I'm not sure.